cmd/atlas/atlas.go

```diff
@@ -99,21 +94,49 @@ func createConfigFromMongoCLIConfig() {
 	_, err = io.Copy(out, in)
 	if err != nil {
 		log.Printf("There was an error generating %s: %v", atlasConfigPath, err)
+		return
 	}
 
 	_, _ = fmt.Fprintf(os.Stderr, `AtlasCLI has copied your MongoCLI configuration to: %s
 
 `, atlasConfigPath)
 }
 
-func mongoCLIConfigPath() string {
+func mongoCLIConfigPaths() (configPath, oldConfigPath string) {
+	configPath = ""
+	oldConfigPath = ""
```

Comment:
[nit] you can remove these vars as they are already defined

Reply:
I'll still need it since I'll move this to another method now. Please, LMK if you have another suggestion in the next rev